src/App.js

```diff
@@ -1,21 +1,12 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import Main from './main'
+import Header from './header'
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
-}
+const App = () => (
```

Comment:
Right now it doesn't need to be a class since it would only have a `render()` method, which would only return jsx that you see here. This will most likely become a class in future, but a simple function is enough for this use.